client/lib/plans/features-list.tsx

```diff
@@ -518,34 +519,52 @@ export const FEATURES_LIST: FeatureList = {
 
 	[ FEATURE_PREMIUM_THEMES ]: {
 		getSlug: () => FEATURE_PREMIUM_THEMES,
-		getTitle: () => {
+		getTitle: () => i18n.translate( 'Premium themes' ),
+		getIcon: () => <img src={ Theme2Image } alt={ i18n.translate( 'Premium themes' ) } />,
+		getDescription: () => i18n.translate( 'Switch between a collection of premium design themes.' ),
+		getConditionalTitle: ( planSlug ) => {
+			if ( ! planSlug ) {
+				return '';
+			}
 			const localeSlug = i18n.getLocaleSlug();
-			const shouldShowNewString =
-				( localeSlug && config< string >( 'english_locales' ).includes( localeSlug ) ) ||
-				i18n.hasTranslation( 'Unlimited premium themes' );
-
-			return shouldShowNewString
-				? i18n.translate( 'Unlimited premium themes' )
-				: i18n.translate( 'Premium themes' );
+			const shouldShowInPersonalPlan =
+				config.isEnabled( 'themes/tiers' ) &&
+				( ( localeSlug && config< string >( 'english_locales' ).includes( localeSlug ) ) ||
```

Comment:
The locale check was implemented to either show the old or new plan name depending on if it was translated for the locale (this was part of the general plan rename project).
Since this is behind a feature flag, and the translation is new. Do we need the locale check here?

Reply:
I added it just in case we enable the flag production before the translations have been completed.